IFC4x3/Properties/n/NumberOfBoreholes_1DBeislK1CbfTPnPGztE29/Documentation.md

```diff
@@ -0,0 +1 @@
+Number of boreholes used for dewatering.
```

Comment:
Too specific for a generic property name.

Reply:
A proposal would have been nice ;-) What about just shortening it to "Number of boreholes" and assume that the Pset provides the context?

Reply:
Done!